cpp/src/io/json/json_tree.cu

```diff
@@ -254,12 +308,423 @@ tree_meta_t get_tree_representation(device_span<PdaTokenT const> tokens,
                                              is_node);
   CUDF_EXPECTS(parent_node_ids_end - parent_node_ids.begin() == num_nodes,
                "parent node id gather mismatch");
+  nvtxRangePop();
+  nvtxRangePop();  // node_parent_ids
```

Comment:
Do we intend to keep all these NVTX ranges and debug print info in this PR?

Reply:
At this point, yes.  NVTX ranges does not cost anything significant. 
Debug prints are very useful throughout the development. It's hard to put the relevant data to print and order back if we remove them now. We intend to keep it until most feature additions are done.

Reply:
Are you planning on removing these in a separate PR? I guess what bothers me about this is that we will likely be in or near code freeze when this is feature is done which means that removing these debug statements will have to be in the next release.

Reply:
Removed all debug prints and all nvtx ranges.